tests/lib/rules/jsx-closing-bracket-location.js

```diff
@@ -653,6 +653,38 @@ ruleTester.run('jsx-closing-bracket-location', rule, {
     }]
   }, {
     code: [
+      'const Button = function(props) {',
+      '  return (',
+      '    <Button',
+      '      size={size}',
+      '      onClick={onClick}',
+      '                                    >',
+      '      Button Text',
+      '    </Button>',
+      '  );',
+      '};'
+    ].join('\n'),
+    output: [
+      'const Button = function(props) {',
+      '  return (',
+      '    <Button',
+      '      size={size}',
+      '      onClick={onClick}',
+      '      >',
+      '      Button Text',
+      '    </Button>',
+      '  );',
+      '};'
+    ].join('\n'),
+    options: ['props-aligned'],
```

Comment:
can we also add a test for the "line-aligned" option? We might want to add explicit tests for all of them.